.gitattributes

```diff
@@ -4,3 +4,4 @@
 /.gitignore export-ignore
 /.travis.yml export-ignore
 /.scrutinizer.yml export-ignore
+/.codecov.yml export-ignore
```

Comment:
This file is generally called `codecov.yml` (despite the fact that the Codecov docs suggest both filenames will work, only `codecov.yml` actually does)